internal/activity.go

```diff
@@ -39,11 +39,15 @@ import (
 
 type (
 	// ActivityType identifies an activity type.
+	//
+	// Exposed as: activity:Type
```

Comment:
hm Can we make these doc links https://tip.golang.org/doc/comment#doclinks?

Reply:
That way they are navigable

Reply:
yeah, good idea. Unfortunately it's not as clean because we have to import the entire package name, i.e. `go.temporal.io/sdk/temporal.ApplicationError`, but clickable is better